hack/build/repos.bzl

```diff
@@ -1753,8 +1807,9 @@ def go_repositories():
 
     go_repository(
         name = "com_github_grpc_ecosystem_grpc_gateway",
-        build_file_generation = "on",
+        build_file_generation = "off",  # MANUAL EDIT: see https://github.com/bazelbuild/bazel-gazelle/issues/788
```

Comment:
I can't tell how many of these changes are automated. 
This looks like something you've done, but the linked issue is now closed.
So please write a little more about whether this is a workaround and if so, when will it be possible for us to remove the workaround.

Reply:
Ok, I moved this to a separate function & added some comments.